entity-types/infra-kubernetes_cronjob/definition.yml

```diff
@@ -8,82 +8,80 @@ dashboardTemplates:
-- k8s.namespaceName
+  - k8s.cronjobName
+  - k8s.clusterName
+  - k8s.namespaceName
 synthesis:
   rules:
-    # Commenting the below rule until our 2nd release
-    # kube-state-metrics data via opentelemetry prometheusReceiver 
-    #- compositeIdentifier:
-    #    separator: ":"
-    #    attributes:
-    #      - k8s.cluster.name
-    #      - namespace
-    #      - cronjob
-    #  encodeIdentifierInGUID: true
-    #  name: cronjob
-    #  conditions:
+    # kube-state-metrics data via opentelemetry prometheusReceiver
+    - compositeIdentifier:
+        separator: ":"
+        attributes:
+          - k8s.cluster.name
+          - namespace
+          - cronjob
+      encodeIdentifierInGUID: true
+      name: cronjob
+      conditions:
         # kube-state-metrics cronjob prefix
-    #    - attribute: metricName
-    #      prefix: kube_cronjob_
+        - attribute: metricName
+          prefix: kube_cronjob_
         # identifier attributes
-    #    - attribute: cronjob
-    #      present: true
-    #    - attribute: namespace
-    #      present: true
-    #    - attribute: k8s.cluster.name
-    #      present: true
+        - attribute: cronjob
+          present: true
+        - attribute: namespace
+          present: true
+        - attribute: k8s.cluster.name
+          present: true
         # open telemetry
-    #    - attribute: newrelic.source
-    #      value: 'api.metrics.otlp'
+        - attribute: newrelic.source
+          value: "api.metrics.otlp"
         # if service.name is present, it's a service
-    #    - attribute: service.name
-    #      present: false
+        - attribute: service.name
+          present: false
         # value added for test entities only
-    #    - attribute: newrelicOnly
-    #      value: "true"
-    #  tags:
-    #    cronjob:
-    #      entityTagNames: [k8s.cronjobName]
-    #    namespace:
-    #      entityTagNames: [k8s.namespaceName]
-    #    k8s.cluster.name:
-    #      entityTagNames: [k8s.clusterName]
-    #    schedule:
-    #      entityTagNames: [k8s.cronjob.schedule]
-    
-    # Commenting the below rule until our 2nd release
-    # Infrastructure event data via opentelemetry 
-    #- compositeIdentifier:
-    #    separator: ":"
-    #    attributes:
-    #      - k8s.cluster.name
-    #      - k8.namespace.name
-    #      - k8s.object.name
-    #  encodeIdentifierInGUID: true
-    #  name: k8s.object.name
-    #  conditions:
-    #    - attribute: event.name
-    #      value: InfrastructureEvent
-    #    - attribute: k8s.object.kind
-    #      value: CronJob
+        - attribute: newrelicOnly
+          value: "true"
+      tags:
+        cronjob:
+          entityTagNames: [k8s.cronjobName]
+        namespace:
+          entityTagNames: [k8s.namespaceName]
+        k8s.cluster.name:
+          entityTagNames: [k8s.clusterName]
+        schedule:
+          entityTagNames: [k8s.cronjob.schedule]
+
+    # Infrastructure event data via opentelemetry
```

Comment:
Do we need to enable entity synthesis from events if the same entity is supposed to be created by the K8s OTel metrics?